tasks/zadanie_15.php

```diff
@@ -17,4 +22,25 @@
 
 $shops = $params[0]; // tej linijki nie ruszamy :)
 
-echo "Najlepsza oferta: Sklep C — 1190 zł";
\ No newline at end of file
+$lowestprice = 10000000;
+
+foreach ($shops as $key => $value){
+	
+  if (is_array($value)){
+    
+    $store = "" ;
+    $price = 0 ;
+
+    foreach ($value as $key2 => $value2){
+
+      if ($key2 == "price"){
+        $price = $value2;
+        }elseif($key2 == "store"){
+      	$store = $value2;
+        }
+      } if ($lowestprice > $price){
+  	$lowestprice = $price;
+  	$shopname = $store;
+      }
+    } 
+  }echo "Najlepsza oferta: " . $shopname . " - " . $lowestprice . " zł" ;
```

Comment:
ℹ️ dobrą zasadą dla nowego typu instrukcji jest dawanie ich w nowej linii 😉 

if ...else .. jest jakby jedną instrukcją i tutaj robimy to tak:

if () {
    //...
} else {
   // ...
}

echo jest inną instrukcją i dlatego powinna trafić do nowej linii 😉 
Ogólnie dla komputera to bez różnicy czy wszystko być miała w jednej linijce czy w milionie, ale dla innych ludzi jak przyjdzie czytać kod może zrobić różnicę 😄 

Ale to no tylko taka dobra rada dla zwiększenia czytelności na przyszłość 😉 
```suggestion
  }
  echo "Najlepsza oferta: " . $shopname . " - " . $lowestprice . " zł" ;
```